<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowStatisticsMassageSale</class>
 <widget class="QMainWindow" name="WindowStatisticsMassageSale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>養生館銷售統計</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_income">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>銷售明細</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget_massage_sale">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <column>
               <property name="text">
                <string>massage_case_key</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>消費日期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>顧客編號</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>姓名</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>產品名稱</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>數量</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>單位</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>單價</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>小計</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>抽成率</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>抽成金</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>銷售師父</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="verticalFrame">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>銷售總覽</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidget_sale_summary">
               <property name="maximumSize">
                <size>
                 <width>600</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <column>
                <property name="text">
                 <string>產品名稱</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>數量</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>金額</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>抽成</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_chart"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="action_export_excel">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-export.svg</normaloff>../icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>匯出Excel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
